### DISCUSSION TITLE
Can I use a prompt with this whisper version?

### DISCUSSION THREAD

**Discussion by DavidFarago** [openai/whisper#1159] [Q&A]
I would like to use this whisper version (run locally), but give it a prompt as in https://platform.openai.com/docs/api-reference/audio/create#audio/create-prompt. Is this possible?

I need the prompt to get the output in a specific format, and to give it specific medical German terms without the need to fine-tune those terms in various settings (gender, noise, ...).

I need to run whisper locally for data protection reasons.

**Comment by glangford** [comment]
After you have installed whisper locally, `whisper --help` will display the different command line options. One of them is
```
--initial_prompt INITIAL_PROMPT
                        optional text to provide as a prompt for the first window. (default: None)
```
In Python, see
* https://github.com/openai/whisper/discussions/355
